src/panels/lovelace/cards/hui-entities-gone-wild.js

```diff
@@ -0,0 +1,130 @@
+import { html } from '@polymer/polymer/lib/utils/html-tag.js';
+import { PolymerElement } from '@polymer/polymer/polymer-element.js';
+
+import '../../../components/entity/state-badge.js';
+import '../../../components/ha-card.js';
+
+import computeStateDisplay from '../../../common/entity/compute_state_display.js';
+import computeStateName from '../../../common/entity/compute_state_name.js';
+
+import EventsMixin from '../../../mixins/events-mixin.js';
+import LocalizeMixin from '../../../mixins/localize-mixin.js';
+
+/*
+ * @appliesMixin EventsMixin
+ * @appliesMixin LocalizeMixin
+ */
+class HuiEntitiesGoneWildCard extends LocalizeMixin(EventsMixin(PolymerElement)) {
+  static get template() {
+    return html`
+    <style>
+      ha-card {
+        padding: 16px;
```

Comment:
We should drop the padding. Let it be the choice of user if they want to add white space to image?

Reply:
Hmm now I'm not sure about this anymore..